site/src/scaffold/styles/layout-buttons.scss

```diff
@@ -0,0 +1,6 @@
+$layouts: 'single', 'holy-grail', 'sidecar-left', 'sidecar-right';
+
```

Comment:
Seems this line can be removed now, too:

```suggestion
```

Reply:
Good shout! Removed